### DISCUSSION TITLE
What is your favorite AI device? Share your thoughts with me.

### DISCUSSION THREAD

**Discussion by oyewaleoyebode** [NAIST-SE/AttractingContributors#1117] [2024 Attraction Contributions]
My favorite AI device is the Amazon Echo. Its voice assistant, Alexa, is incredibly helpful for managing my daily tasks, controlling smart home devices, and even providing entertainment.
Please click on the following link to contribute to my project [My Link](https://github.com/oyewaleoyebode/AttractingContributors/blob/main/README.md)

**Comment by Samabena** [comment]
Hello @oyewaleoyebode  , I recently made a contribution to your project , could you please contribute to my as well ? Following is the link to [my repository](https://github.com/Samabena/sighting) . Thanks in advance.

**Comment by kohanaist** [comment]
I made a contribution to your project. Please confirm it. And, my repository is https://github.com/kohanaist/AttractingContributors-technical-or-specialized-book

**Comment by Ashmari** [comment]
Hi, I made a contribution.
I would be happy if you could also send a pull request to my repository!

https://github.com/Ashmari/AttractingContributors_BookVoyage.git

**Comment by ShunHattori** [comment]
I just made a PR on your repo.
Could you contribute to my repo? Thanks in advance.
[https://github.com/ShunHattori/AttractingContributors_EssentialGadget.git](https://github.com/ShunHattori/AttractingContributors_EssentialGadget.git)
Discussion link: [#1148](https://github.com/NAIST-SE/AttractingContributors/discussions/1148)

**Comment by kohanaist** [comment]
Dear ShunHattori,

Sorry for late reply.

I didn't receive your Pull Request.
So, can you make a Pull Request on my repo?
https://github.com/kohanaist/AttractingContributors-technical-or-specialized-book

And, I have contributed to your repo.
Please confirm it.


2024-06-27 23:20 に ShunHattori さんは書きました:

**Comment by anzhitingg** [comment]
Hello！
I have just made a PR on your repo. 
Could you contribute to mine? 
Here : https://github.com/anzhitingg/AttractingContributors-MBTI
Thank you very much！
